library/classes/rulesets/PQRS/reports/PQRS_0007/InitialPatientPopulation2.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * PQRS Measure 0007 -- Initial Patient Population 2
+ *
+ * Copyright (C) 2016      Suncoast Connection
+ * @package PQRS_Gateway 
+ * @link    http://suncoastconnection.com
+ * @author  Bryan lee <bryan@suncoastconnection.com>
+ * @author  Art Eaton <art@suncoastconnection.com>
+ */
+ 
+class PQRS_0007_InitialPatientPopulation2 extends PQRSFilter
+{
+    public function getTitle() 
+    {
+        return "Initial Patient Population 2";
```

Comment:
If xlt is wanted, it should happen when viewing a report.  These are values that go into the report_itemized anyway, not viewed in the report.